libbpf-rs/tests/test.rs

```diff
@@ -1254,3 +1254,22 @@ fn test_object_perf_buffer_raw() {
 
     assert!(found_cookie);
 }
+
+/// Check that we can get the map pin path from map object.
+#[test]
+fn test_map_get_pin_path() {
```

Comment:
Let's include this functionality into the existing `test_object_loading_pinned_map_from_path` test instead of creating a new one? It seems to fit fine in there, I'd say. Can we also include a call to `is_pinned`, please?